docs/src/modules/java/pages/grpc-endpoints.adoc

```diff
@@ -5,32 +5,34 @@ include::ROOT:partial$include.adoc[]
 An Endpoint is a component that creates an externally accessible API. Endpoints are how you expose your services to the outside world. Two different types of endpoints are available: HTTP endpoints and gRPC endpoints. In this page, we will focus on gRPC endpoints.
 
 gRPC Endpoint components make it possible to conveniently define public APIs accepting and responding in protobuf --
-a binary, typed protocol which is designed to handle evolution of a service over time.
+a binary, typed protocol which is designed to handle evolution of a service over time. They can be used to define both unary and streaming RPCs. For more information on gRPC, see https://grpc.io.
 
 == Basics ==
 To define a gRPC Endpoint component, you start by defining a `.proto` file that defines the service and its messages
 in `src/main/proto` of the project.
 
 [source,protobuf]
-.{sample-base-url}/grpc-shopping-cart-quickstart/src/main/proto/shoppingcart/api/shopping_cart_endpoint.proto[shopping_cart_endpoint.proto]
+.{sample-base-url}/event-sourced-customer-registry/src/main/proto/customer/api/customer_grpc_endpoint.proto[customer_grpc_endpoint.proto]
 ----
-include::example$grpc-shopping-cart-quickstart/src/main/proto/shoppingcart/api/shopping_cart_endpoint.proto[tag=minimal]
+include::example$event-sourced-customer-registry/src/main/proto/customer/api/customer_grpc_endpoint.proto[tags=minimal;get]
 ----
 
-When compiling the project, a Java interface for the service is generated `shoppingcart.api.proto.ShoppingCartEndpoint`. Define a class implementing this interface in the `api` package of your project
+When compiling the project, a Java interface for the service is generated `customer.api.proto.CustomerGrpcEndpoint`. Define a class implementing this interface in the `api` package of your project
 and annotate the class with `@GrpcEndpoint`:
 
 [source,java]
 ----
 @GrpcEndpoint // <1>
-public class ShoppingCartEndpointImpl implements ShoppingCartEndpoint {
+public class CustomerGrpcEndpointImpl implements CustomerGrpcEndpoint {
 
   @Override
-  public CompletionStage<ShoppingCartEndpointOuterClass.ShoppingCart> getCart(GetCartRequest in) {
-    var protoCartBuilder = ShoppingCartEndpointOuterClass.ShoppingCart.newBuilder(); // <2>
-    protoCartBuilder.setCartId(in.getCartId());
-    protoCartBuilder.setCheckedOut(false);
-    return CompletableFuture.completedFuture(protoCartBuilder.build()); // <3>
+  public CompletionStage<Customer> getCustomer(GetCustomerRequest in) {
+    // dummy implementation with hardcoded values
+    var customer = Customer.newBuilder() // <2>
+        .setName("Alice")
+        .setEmail("alice@email.com")
+        .build();
+    return CompletableFuture.completedFuture(customer); // <3>
```

Comment:
Would be better if it was actually compiled to check it is correct, so maybe pull it out to doc-snippets sample?